src/gam.py

```diff
@@ -151,11 +151,32 @@ def csvFieldErrorExit(fieldName, fieldNames):
 def printLine(message):
   sys.stdout.write(message+u'\n')
 #
+def getBoolean(item, i):
```

Comment:
Please don't pass i around like this, it's only going to make cleaning up our sys.argv usage much harder one day. Instead just pass the necessary sys.argv values as needed. That way calling the function doesn't depend on position of arguments.